sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/v2ResolutionPlans.scala

```diff
@@ -63,14 +73,14 @@ case class ResolvedNamespace(catalog: SupportsNamespaces, namespace: Seq[String]
  */
 case class ResolvedTable(catalog: TableCatalog, identifier: Identifier, table: Table)
   extends LeafNode {
-  override def output: Seq[Attribute] = Nil
+  override lazy val output: Seq[Attribute] = table.schema().toAttributes
```

Comment:
do we still need this change after https://github.com/apache/spark/pull/27243#discussion_r368376668 ?

Reply:
Thanks for catching it. Reverted.